src/lib/y2storage/planned/can_be_formatted.rb

```diff
@@ -245,10 +245,11 @@ def reuse_device!(device)
 
       # @param filesystem [Filesystems::Base]
       def assign_mount_point(filesystem)
-        if mount_point && !mount_point.empty?
-          filesystem.mount_path = mount_point
-          filesystem.mount_point.mount_by = mount_by if mount_by
-        end
+        return unless mount_point
```

Comment:
NP: why not this?

```
return if !mount_point || mount_point.empty?
```

Reply:
well, both is possible and rubocop does not prevent it. I ( so personal taste ) just usually prefer having simple exit criteria. So return if nil and return if empty.

Reply:
Ok, yes, a matter of taste. In my case, I prefer to have only one line breaking the method workflow. And when the condition is too complex (IMHO in this case is easy enough) I tend to create a question method:

```
return if empty_mount_point?(mount_point)
```

But I am also fine with your proposal.